libsaas/services/github/forks.py

```diff
@@ -32,11 +32,17 @@ def update(self, *args, **kwargs):
         raise base.MethodNotSupported()
 
     @base.apimethod
-    def create(self):
+    def create(self, organization=None):
         """
         Fork this repo.
+
+        :var organization: Specify the organization you would
+            like to fork to. If left as `None` repo will fork
+            to current user.
+        :vartype organization: str
         """
-        request = http.Request('POST', self.get_url())
+        params = base.get_params(('organization',), locals())
```

Comment:
You can use `get_params(None, locals())` to make it simpler and more consistent with other methods like this one.